### ISSUE TITLE
Improve CONTRIBUTING.md

### ISSUE THREAD

**Issue by bencooper222** [VandyHacks/vaken#32]
Should include instructions on how to setup dev environment.

**Comment by cktang88** [comment]
I'm going to init it

**Comment by cktang88** [comment]
Dependent on #69

**Comment by bencooper222** [comment]
Now that we have a better idea of how everything but the docker flow is going to work, I'm going to write up part of it based on all the new tools that I've introduced to the repo.

**Comment by cktang88** [comment]
Closing in favor of #288
